index.js

```diff
@@ -96,8 +96,44 @@ class Dynamodb extends Datastore {
         });
     }
 
+    /**
+     * Update a record in the datastore
+     * @param  {Object}   config             Configuration object
+     * @param  {String}   config.table       Table name
+     * @param  {Object}   config.params      Record data
+     * @param  {String}   config.params.id   Unique id. Typically the desired primary key
+     * @param  {Object}   config.params.data The data to update with
+     * @param  {Function} callback           fn(err, data)
+     *                                       err - Error object
+     *                                       data - Data saved in the table
+     */
     update(config, callback) {
-        callback();
+        const id = config.params.id;
+        const userData = config.params.data;
+        const client = this.client[config.table];
+        const updateOptions = {
+            expected: { id }
+        };
+
+        if (!client) {
+            const err = new Error(`Invalid table name "${config.table}"`);
+
+            return callback(err);
+        }
+
+        userData.id = id;
+
+        return client.update(userData, updateOptions, (err, data) => {
+            if (err) {
+                if (err.statusCode === 400) {
```

Comment:
In the future we might want to revisit error handling when the `update` returns 400 from "The conditional request failed" vs. the ID doesn't already exist